docs/edgekv_cli.md

```diff
@@ -292,9 +292,10 @@ List items within a namespace and group
 
 Usage: `akamai edgekv list items <environment> <nameSpace> <groupId>`
 
-| Option | Description |
-| - | - |
-| -h, --help  | Display information on how to use this EdgeKV command |
+| Option | Existence | Description |
+| - | - | - |
+| -h, --help  | Optional | Display information on how to use this EdgeKV command |
+| --maxItems  | Optional | Maximum number of items to return (up to system limit) |
```

Comment:
@ahsankamal  can you ping sarah the text and make sure if its okay ? she usually approves the copy. Also is there a max limit for this, if so we should mention it.

Reply:
can you check with medhat on that. Customers might not know the system limit so its good to leave that part out.

Reply:
Sarah suggested to change it to "Maximum number of items to return per request, up to the system [limits](https://techdocs.akamai.com/edgekv/docs/limits) because limit might change in the future.